Matchfile

```diff
@@ -0,0 +1,10 @@
+# The docs are available on https://docs.fastlane.tools/actions/match
+
+git_url("https://github.com/mapbox/apple-certificates")
+storage_mode("git")
+
+app_identifier("com.mapbox.visionsdk.demo")
+
+# Possible values appstore, adhoc, enterprise or development
```

Comment:
nit:

```suggestion
# Possible values appstore, adhoc, enterprise, development or developer_id
```

?

Reply:
It doesn't seem that other types except for `development` and `appstore` are applicable in this case. I think I'll leave just those two. @dersim-davaod agree?

Reply:
Absolutely :)